liboptv/src/segmentation.c

```diff
@@ -113,6 +113,7 @@ int targ_rec (unsigned char *img, target_par *targ_par, int xmin,
 
                 for (n=0; n<4; n++) {
                     xn = x4[n];  yn = y4[n];
+                    if ((xn < xmax) && (yn < ymax)){
```

Comment:
To reduce the levels of nesting, I prefer something like 
```
if (!(xn < xmax) || !(yn < ymax))
    continue;
```

Reply:
got it. thanks